pkgs/applications/blockchains/clightning/default.nix

```diff
@@ -7,6 +7,7 @@
 , automake
 , gettext
 , libtool
+, lowdown
```

Comment:
You are adding `lowdown` here, but not to the `buildInputs`

Reply:
Done! Just for me to understand: Why could I build the package locally without `lowdown` in the `buildInputs`?

Reply:
The upstream project ships local copy of lowdown which is used when system-wide installed one cannot be found - see https://github.com/ElementsProject/lightning/tree/master/external

Reply:
Oh... Thanks!